commands/account/info.ts

```diff
@@ -1,41 +1,47 @@
-// @ts-nocheck
-const { logger } = require('@hubspot/local-dev-lib/logger');
-const { getAccountConfig } = require('@hubspot/local-dev-lib/config');
-const { getAccessToken } = require('@hubspot/local-dev-lib/personalAccessKey');
-const { addConfigOptions } = require('../../lib/commonOpts');
-const { i18n } = require('../../lib/lang');
-const { getTableContents } = require('../../lib/ui/table');
+import { Argv } from 'yargs';
+import { logger } from '@hubspot/local-dev-lib/logger';
+import { getAccountConfig } from '@hubspot/local-dev-lib/config';
+import { getAccessToken } from '@hubspot/local-dev-lib/personalAccessKey';
+import { addConfigOptions } from '../../lib/commonOpts';
+import { i18n } from '../../lib/lang';
+import { getTableContents } from '../../lib/ui/table';
+import { CommonOptions } from '../../types/Yargs';
 
 const i18nKey = 'commands.account.subcommands.info';
-exports.describe = i18n(`${i18nKey}.describe`);
+const describe = i18n(`${i18nKey}.describe`);
 
-exports.command = 'info [account]';
+const command = 'info [account]';
 
-exports.handler = async options => {
+async function handler(options: CommonOptions): Promise<void> {
   const { derivedAccountId } = options;
   const config = getAccountConfig(derivedAccountId);
   // check if the provided account is using a personal access key, if not, show an error
   if (config && config.authType === 'personalaccesskey') {
     const { name, personalAccessKey, env } = config;
+    let scopeGroups: string[][] = [];
 
-    const response = await getAccessToken(
-      personalAccessKey,
-      env,
-      derivedAccountId
-    );
+    if (personalAccessKey) {
```

Comment:
Does this change the behavior of the command?

Reply:
Ah yes, it does. This is actually an unhandled edge case I noticed, but I can fix it in a follow up PR instead of in here. Just because the auth type is personalaccesskey does not mean that the personalaccesskey exists in the config (users could manually remove it).